lib/converters/json_to_jdl_entity_converter.js

```diff
@@ -316,6 +316,9 @@ function addEntityOptionsToJDL(entity, entityName) {
   if (entity.jpaMetamodelFiltering === true) {
     addUnaryOptionToJDL(FILTER, entityName);
   }
+  if (entity.readOnly === true) {
```

Comment:
missing a test here:
  - add in the Employee.json file (used in this converter's test file) this entry "readOnly": true.
  - in the test, l.156, add this:
```javascript
          expect(
            jdlObject
              .getOptions()
              .filter(option => option.name === UnaryOptions.READ_ONLY && option.entityNames.has('Employee')).length
          ).to.equal(1);
```